databroker/core.py

```diff
@@ -911,8 +1027,8 @@ class BlueskyEventStream(DataSourceMixin):
     **kwargs :
         Additional keyword arguments are passed through to the base class.
     """
-    container = 'databroker-xarray'
-    name = 'bluesky-event-stream'
+   # container = 'databroker-xarray'
```

Comment:
No need to keep around the old version of this line as a comment.

Reply:
updated